sdk/monitor/monitor-opentelemetry/src/metrics/standardMetrics.ts

```diff
@@ -144,6 +145,7 @@ export class StandardMetrics {
    * @internal
    */
   public recordLog(logRecord: LogRecord): void {
+    logRecord.setAttribute("operation/synthetic", isSyntheticLoad(logRecord));
```

Comment:
Do you want to add this attribute always even if is not synthetic load?, it would be better to only add it when present

Reply:
If we didn't add the metric on synthetic loads wouldn't the breakdown on the property basically just be: "True" or undefined? Seems like that would conflict with the spec and make the "False" value redundant.

Reply:
I mean do not add extra information the HTTP request if the data is not relevant or it has no use, like for regular not synthetic traffic

Reply:
Got you, updated the method to only attach the property if it's necessary.